<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>9</y>
      <width>101</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="skeletImageButton">
       <property name="text">
        <string>Skelet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pivotsButton">
       <property name="text">
        <string>Pivots</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Rotate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveImageButton">
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePolyButton">
       <property name="text">
        <string>Save Poly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openImageButton">
       <property name="text">
        <string>Open Image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="curWidget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window.</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOne_more_window">
    <property name="title">
     <string>One more window</string>
    </property>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuOne_more_window"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
